### ISSUE TITLE
Cleanup Code

### ISSUE THREAD

**Issue by ChristianVaughn** [cvaughn55/C2F2L#3]
Html is missing accessibility features, javascript and css are also unnecessarily long. Should go through and rewrite most of it to improve accessibility and performance on desktop and mobile clients.
